opendap/src/main/java/opendap/servlet/www/jscriptCore.java

```diff
@@ -140,20 +140,12 @@
             + "\"that appears in the text boxes below the variable. \" +\n"
             + "\"<li>To get ASCII or binary values for the variables you've selected, \" +\n"
             + "\"click on the <em>Get ASCII</em> or <em>Get Binary</em> buttons. \" +\n"
-            + "//  \"<li>To use the <em>Send to Program</em> feature you must install a web \" +\n"
-            + "//  \"helper application that can route the DODS URL built by this form to a \" +\n"
-            + "//  \"client. Look on the <a \" +\n"
-            + "//  \"href=\\\"http://unidata.ucar.edu/packages/dods/\\\" target=\\\"DODS\\\"> \" +\n"
-            + "//  \"DODS home page (http://unidata.ucar.edu/packages/dods/)</a> \" +\n"
-            + "//  \"for a suitable client. \" +\n"
-            + "//  \"</ul>\" +\n"
             + "\"Note that the URL displayed in the <em>DataURL</em> field is updated \" +\n"
             + "\"as you select and/or constrain variables. The URL in this field can be \" +\n"
             + "\"cut and pasted in various DODS clients such as the Matlab and IDL \" +\n"

```

Comment:
This "DODS" should be changed to "OPeNDAP" as well.

Otherwise, looks good to me.

Reply:
Good catch!